data/words/famous_inspirational-quotes.json

```
@@ -0,0 +1,1311 @@
+{
+    "description":"Inspirational quotes from famous personalities across the globe and over generation.",
+    "foot-note":"While adding the quoted from various sources, caution of duplicate quotes have been taken. If duplicstion is found then please update and open a pr",
```

Typos:
```suggestion
    "foot-note":"While adding the quoted from various sources, caution of duplicate quotes have been taken. If duplication is found then please update and open a PR",
```

Although we could probably remove the last sentence because I think that applies to everything in Corpora.